src/ProgressSliderDescription.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import "./description.css";
+
+export default function ProgressSliderDescription(props) {
+  debugger;
+  var exercise = props.exercise;
+  var state = "";
+  const getStep = index => {
```

Comment:
The logic here looks good. However I would prefer to write it like this:
```
const getStep = index => {
	if (index === 0) {
		return "warmup";
	}
	
	if (index % 2 === 1) {
		return "run";
	}
	
	return "walk";
}
```

Also please keep the indentation clean. E.g after `else` there should be a space then `{ `.